content/theme/characters/archetypes/republic/senate-guard.md

```diff
@@ -2,5 +2,39 @@
 title: "Senate Guard"
 date: 2020-11-14T18:42:55-08:00
 draft: false
+layout: archetype 
+
 ---
```

Comment:
Remove this line
Also spacing on the attributes and skills is still 1 space than it should be per line. You have an indent of 2 spaces, should be 1.